python/cugraph/structure/graph.py

```diff
@@ -72,6 +76,14 @@ def __init__(self, symmetrized=False, bipartite=False, multi=False,
         self.edgelist = None
         self.adjlist = None
         self.transposedadjlist = None
+        if m_graph is not None:
+            if edge_attr is None:
+                raise Exception('edge_attr not provided')
```

Comment:
It should be ok to have graphs without edge attributes / weights. Several algos don't use weights at all.

Reply:
fixed with better exception throwing and removing edge_attr constraint for graph constructor with multigraph

Reply:
Added test to test the feature